<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrecheckImportDialog</class>
 <widget class="QDialog" name="PrecheckImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="PerformLabel">
     <property name="text">
      <string notr="true">PerformLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="ProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="MessagesTextEdit">
     <property name="documentTitle">
      <string notr="true"/>
     </property>
     <property name="cursorWidth">
      <number>0</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
